scripts/release/install.ps1

```diff
@@ -68,7 +128,8 @@ function Check-Sha256Sum {
     $expected = Get-Content "$file_stem.sha256"
 
     foreach ($line in $expected) {
-        $line -match '(\S+)\s*\*?(.*)'
+        # Silence the output from the match, otherwise it prints `True` or `False`
+        $null = $line -match '(\S+)\s*\*?(.*)'
```

Comment:
PowerShell amazes me, similar to how a scientist would be interested in a yet unknown animal species. I don't want to really roast PowerShell, but it just sounds so chaotic.

Reply:
Yeah, there is so much weird stuff here, but I'm sure everything has its explanation and even good reasoning that no longer makes sense in the modern era, but could be at hype years ago. E.g. I suppose powershell was born from the OOP hype with all the structured data, try/catch and dot net introp. That sounds cool, but also useless for the kind of the tasks it is used to day, which are probably "install program X, and run it, and be as thin as possible, because program X is actually cross-platform while you aren't :-1: "